app/views/content/events/get-the-most-from-events/_content.html.erb

```diff
@@ -110,6 +110,18 @@
       <h3>Event staff</h3>
       <p> Our friendly team will be there to welcome you at the door. Ask them about the event, where to find the experts, and where you can get overseas qualifications checked.</p>
 
+<h3>Hear from attendees at a Get Into Teaching events</h3>
+<p>Watch our video to see how guests at a Get Into Teaching event have found help and advice on training to teach.</p>
+ 
+  <section class="col col-full-content youtube-videos">
```

Comment:
@martyn-w When playing the video on this page, I can't interact with the controls (eg pause)?

However, when playing it on /about-get-into-teaching-events, I can interact with the controls (which is what I'd expect from an accessibility perspective)?

Can you please have a look into why?

Reply:
Ignore this - no longer an issue!